### PR TITLE
advanced docs: 1.5 kernel execution configuration

### PR DESCRIPTION
**Pull request by cscheid**
To be merged when https://github.com/quarto-dev/quarto-cli/pull/9130 lands.

### CODE REVIEW

File: docs/advanced/jupyter/kernel-execution.qmd [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+In order to make the implementation of these features easier for writers of custom kernels, we have made available a fork of Jupyter's `echo_kernel`. 
+This example kernel is meant to showcase the basics of a custom kernel implementation, and our fork adds the setup cell functionality.
+See the [source for more](https://github.com/quarto-dev/quarto_echo_kernel).
+
+:::
+
+## Daemonization
+
+Quarto offers built-in support for `julia` and `python` kernels by default, see the docs for more ([Julia](https://quarto.org/docs/computations/julia.html#kernel-daemon), [Python](https://quarto.org/docs/computations/python.html#kernel-daemon)).
```
Comment (mcanouil) [inline docs/advanced/jupyter/kernel-execution.qmd:72]:
```suggestion
Quarto offers built-in support for `julia` and `python` kernels by default, see the docs for more ([Julia](/docs/computations/julia.qmd#kernel-daemon), [Python](/docs/computations/python.qmd#kernel-daemon)).
```

File: docs/advanced/jupyter/kernel-execution.qmd [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+### Adding support to a Jupyter kernel
+
+{{< include /docs/prerelease/1.5/_pre-release-feature.qmd >}}
+
+Arbitrary Jupyter kernels can indicate support for daemons. To do so, provide a `quarto_setup_cell` file in the kernelspec directory so that setup cells can be executed, and ensure that the execution of the setup cell returns an output result with metadata indicating support for running as a daemon. The supported metadata options are:
+
+- `daemonize`: if a kernel returns `true` when executing Quarto's setup cell, then Quarto will maintain a persistent kernel. The daemon options are the same as in the [documentation above]([Python](https://quarto.org/docs/computations/python.html#kernel-daemon)).
```
Comment (mcanouil) [inline docs/advanced/jupyter/kernel-execution.qmd:80]:
```suggestion
- `daemonize`: if a kernel returns `true` when executing Quarto's setup cell, then Quarto will maintain a persistent kernel. The daemon options are the same as in the [documentation above]([Python](/docs/computations/python.qmd#kernel-daemon)).
```

**Review by mcanouil — COMMENTED** [review]
Update internal links to be actual internal links, see commit suggestions in the review.

Following: https://github.com/quarto-dev/quarto-web/pull/1031

File: docs/advanced/jupyter/kernel-execution.qmd (hunk 2 above)
Comment (cscheid) [reply docs/advanced/jupyter/kernel-execution.qmd:80]: Sure, but why?

Comment (mcanouil) [reply docs/advanced/jupyter/kernel-execution.qmd]: Mostly for consistency and make sure internal links are internal, thus not using the apex.

The "html" to "qmd" is more of a long run thought, i.e., the feature request to have the documentation as PDF.

Comment (mcanouil) [reply docs/advanced/jupyter/kernel-execution.qmd]: Related open discussion: https://github.com/quarto-dev/quarto-cli/issues/9099

Comment (cscheid) [reply docs/advanced/jupyter/kernel-execution.qmd]: > thus not using the apex

I don't understand what you mean here.

> The "html" to "qmd" is more of a long run thought, i.e., the feature request to have the documentation as PDF

Ok, this makes good sense, thanks.

Comment (mcanouil) [reply docs/advanced/jupyter/kernel-execution.qmd]: The `https://quarto.org` has no reason to be there for internal links.
The apex/domain should come from <https://github.com/quarto-dev/quarto-web/blob/main/_quarto.yml#L16>.
At least that is how it is tell/implied in the documentation, so we should follow this rule.

**Pull request merged**